doc/rpc/pathfinder_rpc_api.json

```diff
@@ -295,6 +295,24 @@
             "PROOF_MISSING": {
                 "code": 10000,
                 "message": "Merkle trie proof is not available"
+            },
+            "SUBSCRIPTION_TXN_HASH_NOT_FOUND": {
```

Comment:
This should move to the pathfinder ws spec, and also needs the transaction subscription documentation added there.